zerver/openapi/zulip.yaml

```diff
@@ -16179,3 +16476,15 @@ components:
         type: string
       example: https://github.com/zulip/zulip/issues/%(id)s
       required: true
+    DirectMemberOnly:
+      name: direct_member_only
+      in: query
+      description: |
+        Whether to consider only the direct members of user group and not members
+        of its subgroups. Default value is `False`.
+
+        **Changes**: New in Zulip 6.0 (feature level 127).
```

Comment:
I removed the changes entry, since AFAIK this only appears on endpoints that are new with this feature level.